.eslintrc.js

```diff
@@ -49,6 +49,21 @@ module.exports = {
         'import/no-extraneous-dependencies': 'off',
       },
     },
+    {
+      files: ['scripts/**/*'],
+      rules: {
+        'no-void': 'off',
+        'no-console': 'off',
+        'no-process-exit': 'off',
+        'functional/no-expression-statement': 'off',
+        '@typescript-eslint/naming-convention': 'off',
+        'import/no-extraneous-dependencies': 'off',
```

Comment:
Why do we need this rule?

Reply:
It's not necessary, I copy and pasted the override for `test/**/*` and it was in there, I forgot to remove it.